test/webapi/test_jobs.py

```diff
@@ -11,22 +11,22 @@
 from ecs_scheduler.webapi.jobs import Jobs, Job, require_json_content_type
 
 
-@patch('flask.request')
+@patch('ecs_scheduler.webapi.jobs.flask')
```

Comment:
Flask cannot be patched directly ever since around the 1.0 release for whatever reason